logging.go

```diff
@@ -64,6 +64,7 @@ func (a *App) makeConfigLogger() (timber.ConfigLogger, bool) {
 			// Carry on with stdout logging, but remember to mention it
 			fellbackToCWD = true
 			a.logDir = "."
+			newWriter.Close()
```

Comment:
It might be better to move the creation of the filewriter into the else branch? The error handling is pretty messy here...